docs/scripts/api/utils.ts

```diff
@@ -1,8 +1,8 @@
 import { Annotation } from 'doctrine';
 import kebabCase from 'lodash/kebabCase';
 import * as prettier from 'prettier';
-import * as fse from 'fs-extra';
 import { Symbol, isPropertySignature, isExportSpecifier, TypeFormatFlags } from 'typescript';
+import fs from 'node:fs';
```

Comment:
I imported from `node:fs`, but we seem to import from `fs` directly. I can change that if needed.

Reply:
I'm adding https://github.com/mui/mui-public/issues/476